src/base/detoken.lua

```diff
@@ -49,6 +49,7 @@
 			-- convert the token into a function to execute
 			local func, err = loadstring("return " .. token)
 			if not func then
+				print("load error:", err)
```

Comment:
no... that would abort immediately, and not go back to the caller (detoken) which does the error instead but with a little more context...